lib/bcoin-browser.js

```diff
@@ -89,6 +89,7 @@ bcoin.node = require('./node');
 bcoin.Node = require('./node/node');
 bcoin.FullNode = require('./node/fullnode');
 bcoin.SPVNode = require('./node/spvnode');
+bcoin.Neutrino = require('./node/neutrino');
```

Comment:
@masterchief164 Do we need to add neutrino to bcoin-browser? AFAIK, bcoin-browser is obselete.